### PR TITLE
Support large scale jittering

### PR DESCRIPTION
**Pull request by ZwwWayne**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Several configs are added.
As the baseline also requires deeper RPN head with two shared convolutional layers, RPNHead is modified without BC-breaking.
Relies on open-mmlab/mmcv#1205

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Handled to be compatible with the original case.

## Use cases (Optional)

See configs

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet/core/mask/structures.py [hunk 1]
```diff
@@ -529,6 +529,19 @@ def random(cls,
...
+    def get_bboxes(self):
+        num_masks = len(self)
+        boxes = np.zeros((num_masks, 4), dtype=np.float32)
+        x_any = self.masks.any(axis=1)
+        y_any = self.masks.any(axis=2)
+        for idx in range(num_masks):
+            x = np.where(x_any[idx, :])[0]
+            y = np.where(y_any[idx, :])[0]
+            if len(x) > 0 and len(y) > 0:
+                boxes[idx, :] = np.array([x[0], y[0], x[-1] + 1, y[-1] + 1],
```
Comment (hhaAndroid) [inline mmdet/core/mask/structures.py:541]: So do you need to add a comment to explain why you need to +1?

File: mmdet/core/mask/structures.py [hunk 2]
```diff
@@ -1020,6 +1033,22 @@ def _order_vertices(verts):
         self = cls(masks, height, width)
         return self
 
+    def get_bboxes(self):
+        num_masks = len(self)
+        boxes = np.zeros((num_masks, 4), dtype=np.float32)
+        for idx, poly_per_obj in enumerate(self.masks):
+            xy_min = np.array([self.width * 2, self.height * 2],
```
Comment (hhaAndroid) [inline mmdet/core/mask/structures.py:1040]: I don't understand why *2？

File: mmdet/datasets/pipelines/transforms.py [hunk 3]
```diff
@@ -733,6 +733,7 @@ def __init__(self,
                  crop_size,
                  crop_type='absolute',
                  allow_negative_crop=False,
+                 recompute_bbox=False,
```
Comment (hhaAndroid) [inline mmdet/datasets/pipelines/transforms.py:736]: Docstr is missing

File: mmdet/models/dense_heads/rpn_head.py [hunk 4]
```diff
@@ -23,14 +24,32 @@ class RPNHead(AnchorHead):
     def __init__(self,
                  in_channels,
                  init_cfg=dict(type='Normal', layer='Conv2d', std=0.01),
+                 num_convs=1,
```
Comment (hhaAndroid) [inline mmdet/models/dense_heads/rpn_head.py:27]: Missing docstr

File: mmdet/models/dense_heads/rpn_head.py [hunk 5]
```diff
@@ -23,14 +24,32 @@ class RPNHead(AnchorHead):
...
+                    in_channels = self.in_channels
+                else:
+                    in_channels = self.feat_channels
+                rpn_convs.append(
+                    ConvModule(
+                        in_channels,
+                        self.feat_channels,
+                        3,
+                        padding=1,
+                        inplace=False))
```
Comment (hhaAndroid) [inline mmdet/models/dense_heads/rpn_head.py:48]: Why inplace should be set to False？

Comment (hhaAndroid) [reply mmdet/models/dense_heads/rpn_head.py:48]: This way of writing will result in the existence of two Relu ops. Should it be improved?

Comment (ZwwWayne) [reply mmdet/models/dense_heads/rpn_head.py:48]: > This way of writing will result in the existence of two Relu ops. Should it be improved?

should be fine.

File: configs/common/lsj_100e_coco_instance.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+# dataset settings
+dataset_type = 'CocoDataset'
+data_root = 'data/coco/'
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+image_size = (1024, 1024)
+
+# file_client_args = dict(backend='disk')
+# comment out the code below to use different file client
+file_client_args = dict(
```
Comment (hhaAndroid) [inline configs/common/lsj_100e_coco_instance.py:11]: file_client_args = dict(backend='disk')

**Review by hhaAndroid — APPROVED** [review]

**Pull request merged**